diagnostic.md

```diff
@@ -73,14 +86,25 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/$ID \
+--header "Content-Type: application/json" \
+--data '{
+  "name": "French Cruller",
+  "price": "$0.99"
+}'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function(data){
+  return $.ajax({
+    url: 'http://www.example.com/donuts/$ID',
+    method: 'POST'
+    data,
```

Comment:
This will work since you set `data` as a function argument, but we want to see the actual payload being passed in as well so this should be followed by something like `createDonut({ name: 'French Cruller', price: '$0.99' });`.

Another alternative, given by the solution branch, is
```javascript
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```